mapie/classification.py

```diff
@@ -620,6 +622,7 @@ def _fit_and_predict_oof_model(
             )
         if _num_samples(X_val) > 0:
             y_pred_proba = self._predict_oof_model(estimator, X_val)
+            y_pred_proba = self._check_proba_normalized(y_pred_proba)
```

Comment:
Is this line needed if `_check_proba_normalized` is called in `_predict_oof_model` ?

Reply:
Not needed indeed ! Done